Lib/test/test_pydoc.py

```diff
@@ -906,12 +906,15 @@ class A(builtins.object)
  |  ----------------------------------------------------------------------
  |  Data descriptors defined here:
  |
- |  __dict__
- |      dictionary for instance variables
+ |  __dict__%s
  |
- |  __weakref__
- |      list of weak references to the object
-''' % __name__)
+ |  __weakref__%s
+''' % (__name__,
+       '' if MISSING_C_DOCSTRINGS else
+       '\n |      dictionary for instance variables',
+       '' if MISSING_C_DOCSTRINGS else
+       '\n |      list of weak references to the object',
```

Comment:
I would have been okay with not splitting these expressions across two lines each, but keep each item on a (fairly but not excessively) long line by itself. (I guess in my old age I'm less attached to 80 columns. :-)